### DISCUSSION TITLE
Display # of forks under Repositories

### DISCUSSION THREAD

**Discussion by alphasecio** [community/community#54371] [Mobile]
I'd like to see the number of forks, in addition to stars, for each repo under Repositories tab.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.111.1 (98645245)
</details>

**Comment by mattruggio** [comment]
Hey @alphasecio, thanks for the suggestion! I am going to add this to our feature request backlog.
